code/game/objects/items/cards_ids.dm

```diff
@@ -461,6 +461,26 @@ update_label()
 	access = list(ACCESS_SYNDICATE)
 	uses_overlays = FALSE
 
+/obj/item/card/id/syndicate_command/crew_id

```

Comment:
why do this when it inherits everything from parent and the captain one inherits everything but the name and assignment

Reply:
For some reason, I thought this would be a better decision.
I'm actually thinking about adding some extra access, so people with syndicate chameleon ID won't be able to get here since lore-wise it's an old ship.